sig-service-catalog/charter.md

```diff
@@ -0,0 +1,165 @@
+# SIG Service Catalog Charter
+Service Catalog is a Kubernetes extension project that implements the [Open
+Service Broker API](https://www.openservicebrokerapi.org/) (OSBAPI). It allows
+application developers the ability to provision and consume cloud services
+natively from within Kubernetes.
+
+This charter adheres to the conventions described in the [Kubernetes Charter
+README](https://github.com/kubernetes/community/blob/master/committee-steering/governance/README.md).
+
+## Scope
+
+See the [service-catalog entry in sigs.yaml](https://github.com/kubernetes/community/blob/master/sigs.yaml#L1485).
```

Comment:
either point to just the file (not the line number) since the line number can move (and already has), or just remove this sentence since I'm not sure that pointing to the sigs.yaml is really a better source of truth than what's in here.

Reply:
Just link to the README here: https://github.com/kubernetes/community/tree/master/sig-service-catalog

That README is generated from sigs.yaml.

Reply:
Will do, thanks!